docs/Video Streaming for Navigation Apps/Video Streaming - Java/index.md

```diff
@@ -99,4 +111,76 @@ builder.setRPCNotificationListeners(onRPCNotificationListenerMap);
 ```
 
 ### Handling HMI Scaling (RPC v6.0+)
-If the HMI scales the video stream, you will have to handle scaling the projected view, touches and haptic rectangles yourself (this is all handled for you behind the scenes in the `VideoStreamManager`  API). To find out if the HMI scales the video stream, you must for query and check the `VideoStreamingCapability` for the `scale` property. Please check the [Adaptive Interface Capabilities](Displaying a User Interface/Adaptive Interface Capabilities) section for more information on how to query for this property using the system capability manager.  
\ No newline at end of file
+If the HMI scales the video stream, you will have to handle scaling the projected view, touches and haptic rectangles yourself (this is all handled for you behind the scenes in the `VideoStreamManager`  API). To find out if the HMI scales the video stream, you must for query and check the `VideoStreamingCapability` for the `scale` property. Please check the [Adaptive Interface Capabilities](Displaying a User Interface/Adaptive Interface Capabilities) section for more information on how to query for this property using the system capability manager.
+
+### Video Streaming Parameters (SDL v5.1+)
+Starting with SDL version 5.1+ the `VideoStreamingParameters` you provide will automatically be aligned with the `VideoStreamingCapabilities` provided by the HMI.
+If the HMI provides the scale or resolution in the `VideoStreamingCapabilities` the Video stream will use that scale or resolution. Otherwise, the scale or resolution defined in the `VideoStreamingParameters` you provided will be used.
```

Comment:
```suggestion
If the HMI provides the scale or resolution in the `VideoStreamingCapabilities` the video stream will use that scale or resolution. Otherwise, the scale or resolution you defined in the `VideoStreamingParameters` will be used.
```